SwiftUIBasics/Views/PlansView.swift

```diff
@@ -8,8 +8,85 @@
 import SwiftUI
 
 struct PlansView: View {
+    var customColorLightGray = Color(red:240/255, green: 240/255, blue:240/255)
+    var customColorDarkGray = Color(red:62/255, green:63/255, blue: 70/255)
```

Comment:
this is better than having it inline, but for colors is recommended to save it into the assets